src/compiler/emitter.ts

```diff
@@ -3258,6 +3271,12 @@ module ts {
                 tempVariables = undefined;
                 tempParameters = undefined;
                 emitSignatureParameters(node);

```

Comment:
it looks like if you have:

```var v = a => { };```

then we'll emit that as:

```var v = (a) => { };```

Can we make it so we emit this case idiomatically as it was originally written?